### PR TITLE
L2 Assets Frontend: Add token modal

### PR DESCRIPTION
**Pull request by mateuszradomski**
Resolves L2B-2032

In this PR a new way to pick tokens is added, instead of a vulcanic eruption of all tokens possible as a list, the most important tokens now get placed inside a table, separated by CBV/EBV/NMV with icons which is a lot more user friendly.

### CODE REVIEW

**Comment by antooni** [comment]
window resize refreshes the state

**Comment by mateuszradomski** [comment]
> window resize refreshes the state

It's now fixed, the issue was cause by the fact the when the window is resized the entire previousState get's wiped clean, and my `classList.toggle` class were happening over and over. They still do, but right now I force if classes should be added or removed.

File: packages/frontend/src/components/chart/Chart.tsx [hunk 1]
```diff
@@ -120,10 +121,17 @@ export function Chart({
...
+            {hasTvl && (
+              <div className="flex h-[2rem] items-end">
+                <CurrencyControls />
+                {hasDetailedTvl && <TokenControls tokens={tokens} />}
+              </div>
+            )}
             <ScaleControls />
           </div>
-          {hasTvl && <TokenControls tokens={tokens} />}
+          {hasTvl && !hasDetailedTvl && (
```
Comment (antooni) [inline packages/frontend/src/components/chart/Chart.tsx:132]: This flag does not work on staging, the "Regular TVL" shows new modal

Comment (mateuszradomski) [reply packages/frontend/src/components/chart/Chart.tsx:132]: On production it does not

**Review by antooni — APPROVED** [review]

**Pull request merged**